lib/src/modules/auth/domain/interfaces/auth_repository_interface.dart

```diff
@@ -3,11 +3,12 @@ import 'package:dartz/dartz.dart';
 import '../../../../core/infrastructure/datasources/remote/api/base/api_error.dart';
 import '../../../../core/infrastructure/datasources/remote/api/services/auth/models/login_request.dart';
 import '../../infrastructure/models/user_model.dart';
```

Comment:
better to remove this import too

Reply:
Removed `user_model.dart` import
Thanks